apps/mobile/src/components/SafeFontIcon/SafeFontIcon.tsx

```diff
@@ -4,7 +4,7 @@ import { useFonts } from 'expo-font'
 import { IconName } from '@/src/types/iconTypes'
 import { getVariable, useTheme } from 'tamagui'
 
-export const SafeIcon = createIconSetFromIcoMoon(
+const SafeIcon = createIconSetFromIcoMoon(
```

Comment:
It is not necessary to export this. It's only needed here and often I was erroniously importing it.